python/pyspark/ml/feature.py

```diff
@@ -2502,7 +2524,7 @@ def __init__(self, numTopFeatures=50, featuresCol="features", outputCol=None, la
             Param(self, "numTopFeatures",
```

Comment:
This whole line could be removed.  This must be a leftover from when we used to have to specify Params twice for Python.